invest_cal.py

```diff
@@ -19,5 +19,5 @@ def calculator():
     print(stockrate)
     inputs(stockrate)
     
-    
+#this is a new comment  
```

Comment:
This comment provides no value, we should remove this. Also, we will want to remove some of the hardcoded values in this file, as it doesn't make the code extensible.